DEVELOPING.md

```diff
@@ -3,24 +3,27 @@ We appreciate any contributions to the Score-P Python Bindings.
 However, there are a few policies we agreed on and which are relevant for contributions.
 These are detailed below. 
 
-## Fromatting / Codestyle
+## Formatting / Codestyle
 
 Readable and consistent code makes it easy to understand your changes.
-Therefore the GitHub CI system has checks using `clang-format-9` and `flake8` in place.
-Please make sure that these test pass, before making a Pull Request.
-If you do not use the GitHub CI process, you can use the provided `.flake8` and `.clang-format` files, to check your code manually.
```

Comment:
Slight rewording as it might be confusing to some, to make CI tests pass without opening a PR. Also mentioning what tools to use to fix issues locally